utils/conversions.ts

```diff
@@ -43,15 +41,28 @@ import { UserBonus } from "../models/user_bonus";
 import { renameObjectKeys } from "./rename_keys";
 import { camelCaseToSnakeCase, snakeCaseToCamelCase } from "./strings";
 
-// 150 EUR for one member families
-const ONE_FAMILY_MEMBER_AMOUNT = 150 as MaxBonusAmount;
-// 250 EUR for two member families
-const TWO_FAMILY_MEMBERS_AMOUNT = 250 as MaxBonusAmount;
-// 500 EUR for three or more member families
-const THREE_OR_MORE_FAMILY_MEMBERS_AMOUNT = 500 as MaxBonusAmount;
-
-// Max tax benefit is 20% of max bonus amount
-const TAX_BENEFIT_PERCENT = 20;
+const OneFamilyMemberBonus = {
```

Comment:
@gunzip what about this? we make internal logic type safe and completely abstracted from the app and ADE APIs point of view so we can change it whenever we want ?

Reply:
lgtm!

Reply:
what I didn't like in the previous PR was that amount and benefit were not coupled (now they are) and the computation of tax benefit which was useless